modules/raft/src/integrationTest/java/org/apache/ignite/raft/server/counter/GetValueCommand.java

```diff
@@ -18,9 +18,14 @@
 package org.apache.ignite.raft.server.counter;
 
 import org.apache.ignite.internal.raft.ReadCommand;
+import org.apache.ignite.network.annotations.Transferable;
 
 /**
  * Get a value command.
  */
-public class GetValueCommand implements ReadCommand {
+@Transferable(1_000)
```

Comment:
We do not have a single place where all types of messages?

Reply:
No, we don't, I'll do it if you're asking